src/app/code/community/Instamojo/Imojo/controllers/PaymentController.php

```diff
@@ -23,7 +23,7 @@ public function redirectAction()
 
             // Set status to payment pending
             $order->setState(Mage_Sales_Model_Order::STATE_PENDING_PAYMENT, true)->save();
-            $amount = $order-> getBaseGrandTotal();
+            $amount = $order->getGrandTotal();
```

Comment:
Add a comment about why we are using `getGrandTotal()` here instead of `getBaseGrandTotal()`?